desc/compute/_core.py

```diff
@@ -67,7 +67,7 @@ def _rho_r(params, transforms, profiles, data, **kwargs):
     units="~",
     units_long="None",
     description="Radial coordinate, proportional to the square root "
-    + "of the toroidal flux, derivative wrt poloidal coordinate",
+    "of the toroidal flux, derivative wrt poloidal coordinate",
```

Comment:
If I remember correctly, the cause of this lost `+` sign was a script I used to update a mistake I made while copying pasting descriptions. Shouldn't affect anything since the string is concatenated.